.github/workflows/ci.yml

```
@@ -71,31 +71,78 @@ jobs:
+  # as "not built yet" and skipped the only check that mattered. A check that skips itself is
+  # worse than one that is absent, because the green tick says it ran.
+  #
+  # Nothing is guarded now. Every command below runs on a runner with no endpoint, no TTY and no
+  # `~/.hera`, which is precisely the environment the escape-code bug would hide in.
   terminal:
     name: terminal
     runs-on: ubuntu-latest
+    env:
+      # A temporary data directory, so nothing here can touch a real one -- and `HOME` on a runner
+      # is real enough to matter.
+      HERA_HOME: ${{ runner.temp }}/hera-home
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)